vendor/assets/stylesheets/fileupload/jquery.fileupload-ui.css.erb

```diff
@@ -0,0 +1,86 @@
+@charset 'UTF-8';
+/*
+ * jQuery File Upload UI Plugin CSS 6.3
```

Comment:
This seems out of date from the current one https://github.com/blueimp/jQuery-File-Upload/blob/master/css/jquery.fileupload-ui.css